src/ctapipe/io/simteleventsource.py

```diff
@@ -637,10 +637,7 @@ def prepare_subarray_info(self, telescope_descriptions, header):
                     )
                 except ValueError:
                     telescope = unknown_telescope(mirror_area, n_pixels)
-
-                # TODO: switch to warning or even an exception once
-                # we start relying on this.
-                self.log.debug(
+                self.log.warning(
```

Comment:
This will create log.warning for each telescope when analyzing prod5 data. Do we really want to enable this?

Reply:
It should maybe just be at the `info` level, not warning. It is useful to show, since it makes it clear when the user is using an old data file with is lacking metadata.